src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -350,9 +361,9 @@ public static EnergyBalanceInfo ComputeEnergyBalance(IReadOnlyList<OrganelleTemp
     /// </remarks>
     public static Dictionary<Compound, CompoundBalance> ComputeCompoundBalance(
         IEnumerable<OrganelleDefinition> organelles, IBiomeConditions biome, CompoundAmountType amountType,
-        bool requireInputCompoundsInBiome)
+        bool requireInputCompoundsInBiome, Dictionary<Compound, CompoundBalance>? result = null)
```

Comment:
Also same here and multiple methods afterwards: I think the result parameter should be required so that the callsites can then more intelligently be fixed to reuse objects (like is the goal for improving the game allocation counts).